src/jit/block.h

```diff
@@ -438,6 +438,8 @@ struct BasicBlock : private LIR::Range
 #define BBF_CLONED_FINALLY_BEGIN 0x100000000 // First block of a cloned finally region
 #define BBF_CLONED_FINALLY_END 0x200000000   // Last block of a cloned finally region
 
+#define BBF_DOMINATED_BY_NOT_NORMAL_ENTRY 0x400000000 // Full set of dominators contains not normal entry.
+
```

Comment:
Instead of "not normal" can we call it "Exceptional"? e.g., BBF_DOMINATED_BY_EXCEPTIONAL_ENTRY, fgCompDominatedByExceptionalEntryBlocks(), etc.?